spec/definitions/TokenRenewV0.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+properties:
+  user:
+    title: User id
+    description: Id of the user resource
+    type: string
+  enterprise:
+    title: Enterprise id
+    description: Id of the enterprise resource
+    type: string
+    format: uuid
+  authserver_token_id:
```

Comment:
@abdulMuneerM @BindyaBhaskar this api renews the token, so `token` is the renewed token right. What is this `authserver_token_id ` token it for ?

Reply:
Yes, `token` is the renewed token.`authserver_token_id` is the object id in the auth server, this don't need to expose.

Reply:
Removed authserver_token_id from properties.